deploycala.py

```diff
@@ -205,6 +205,8 @@ def main():
     parser.add_argument("-i", "--incremental", action="store_true", dest="incremental",
                         help="do incremental builds, i.e. don't clear the build directory before building, if found")
     parser.add_argument("--noupdate", action="store_true", dest="noupdate", help=argparse.SUPPRESS)
+    parser.add_argument("-u", "--url", nargs=1, default=None, dest="url",
```

Comment:
Agreed. Calamares upstream should be the default to avoid breaking existing behavior.